google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -44,11 +47,17 @@ def _pipeline_cell(args, cell_body):
     bq_pipeline_config = utils.commands.parse_config(
         cell_body, utils.commands.notebook_environment())
     pipeline_spec = _get_pipeline_spec_from_config(bq_pipeline_config)
+    # TODO(rajivpb): This import is a stop-gap for
+    # https://github.com/googledatalab/pydatalab/issues/593
+    import google.datalab.contrib.pipeline._pipeline
     pipeline = google.datalab.contrib.pipeline._pipeline.Pipeline(name, pipeline_spec)
     utils.commands.notebook_environment()[name] = pipeline
 
     # If a composer environment and zone are specified, we deploy to composer
     if 'environment' in args and 'zone' in args:
```

Comment:
I think "'environment' in args and 'zone' in args" is always True. If user does not provide values for them, they will be None but still 'environment' and 'zone' exist in args.

Perhaps you want  "if args['environment'] and args[ 'zone']:".

Reply:
Oh, I didn't know that. Will verify and fix in a different PR mostly because I'm trying to get this in before demo time today.